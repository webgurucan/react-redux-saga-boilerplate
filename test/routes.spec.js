import createRoutes from 'routes';

describe('routes', () => {
  it('should match the snapshot', () => {
    expect(String(createRoutes)).toMatchSnapshot();
  });
});
