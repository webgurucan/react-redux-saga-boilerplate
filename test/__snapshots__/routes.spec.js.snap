exports[`routes should match the snapshot 1`] = `
"function createRoutes(){/* istanbul ignore next */++__cov_YFF48TBiBU$VZkTB0sE976HoIus.f[\'1\'];++__cov_YFF48TBiBU$VZkTB0sE976HoIus.s[\'1\'];
return(
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.Route,/* istanbul ignore next */{path:\'/\',component:/* istanbul ignore next */_App2.default},
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.Route,/* istanbul ignore next */{component:/* istanbul ignore next */_AppPrivate2.default},
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.Route,/* istanbul ignore next */{path:\'private\',component:/* istanbul ignore next */_Logged2.default,onEnter:/* istanbul ignore next */_routerInterceptor.checkStatus}),
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.Route,/* istanbul ignore next */{path:\'*\',component:/* istanbul ignore next */_NotFound2.default})),

/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.Route,/* istanbul ignore next */{component:/* istanbul ignore next */_AppPublic2.default},
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.IndexRoute,/* istanbul ignore next */{component:/* istanbul ignore next */_Home2.default,onEnter:/* istanbul ignore next */_routerInterceptor.checkStatus}),
/* istanbul ignore next */_react2.default.createElement(/* istanbul ignore next */_reactRouter.Route,/* istanbul ignore next */{path:\'*\',component:/* istanbul ignore next */_NotFound2.default}))));



}"
`;
